content/en/synthetics/browser_tests/actions.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Browser Test Actions
+kind: documentation
+description: Record Actions for a Synthetics Browser Test
+aliases:
+- /synthetics/browser_tests
+further_reading:
+- link: "synthetics/browser_tests/advance_options"
+  tag: "Documentation"
+  text: "Learn how to configure advance options for Actions"
+---
+
+## Overview
+
+Actions are a series of steps that you can record for a Browser Test, which you can then edit or build on. Certain actions can also be configured with [advanced options][1].
+
+### Assertion
+
+{{< img src="synthetics/browser_tests/browser_test_assertions.png" alt="Browser Test Assertion"  style="width:40%;">}}
+
+Assertions allow you to check whether an element, some content, or some text is available on the current page. You can also check whether a specific email was sent.
+
+The default timeout for each step is approximately 60 seconds. You can override the timeout with [advance options][2].
```

Comment:
```suggestion
The default timeout for each step is approximately 60 seconds. You can override the timeout with [advanced options][2].
```

Can we move that in the overview section as it's true for all steps and not only assertions?